.envrc

```diff
@@ -0,0 +1,8 @@
+if $(command -v asdf > /dev/null)
```

Comment:
I don't think we want to check this file in.

Reply:
removed